.github/workflows/CI.yaml

```diff
@@ -60,4 +59,59 @@ jobs:
           make fmt
           git diff --exit-code || (echo 'Please run "make fmt" to format code' && exit 1);
           make
-          make test
+          go test -coverprofile=${{ matrix.dir }}.out -covermode=atomic ./...
```

Comment:
Shall we keep the `-race` argument here?

Reply:
LGTM in CI, -race takes too long. But it works in CI.

Reply:
I would add them at next PR.